specification/dartLangSpec.tex

```diff
@@ -28,6 +28,11 @@
 % Significant changes to the specification.
 %
 % 2.4
+% - Change specification of `await` to be type-safe.
+%   The existing specification allowed predictable run-time type errors for
+%   well-typed programs like
+%       FutureOr<Future<S>> ffs = Future<S>.value(s);
+%       Future<S> fs = await ffs;
```

Comment:
This should be added at the end of the 2.4 list, not the beginning.

Also, 5 lines of explanation in the change log for an update is long. How about:

% - Change specification of `await` to be type-safe, preventing unsound constructs like
%       FutureOr<Future<int>> ffs = Future<int>.value(s);
%       Future<int> fs = await ffs;

Reply:
DOne.